src/jit/compiler.h

```diff
@@ -1329,11 +1374,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     DWORD expensiveDebugCheckLevel;
 #endif
 
-#if FEATURE_MULTIREG_RET
+#if FEATURE_MULTIREG_RET 
```

Comment:
Even @adiaaida  is changing this code (both #ifdef and name of the method).  Something to watch out for conflict resolution as it is likely that @adiaaida will checked-in ahead of you.